tools/make/kube.mk

```diff
@@ -4,6 +4,14 @@ ENVTEST_K8S_VERSION ?= 1.24.1
 # For more details, see https://gateway-api.sigs.k8s.io/guides/getting-started/#installing-gateway-api 
 GATEWAY_API_VERSION ?= $(shell go list -m -f '{{.Version}}' sigs.k8s.io/gateway-api)
 
+REGISTRY ?= docker.io/envoyproxy
+# IMAGE is the image URL for build and push image targets.
+IMAGE ?= ${REGISTRY}/gateway-dev
+# REV is the short git sha of latest commit.
+REV=$(shell git rev-parse --short HEAD)
+# Tag is the tag to use for build and push image targets.
+TAG ?= $(REV)
+
```

Comment:
Looks like we already got these vars in `tools/make/image.mk` and `tools/make/common.mk`?